### PR TITLE
[Merged by Bors] - feat: add doc strings in alias command

### PR DESCRIPTION
**Pull request by dwrensha**
Follows the mathlib3 version: https://github.com/leanprover-community/mathlib/blob/1158f6d88df46d30c657ffce6bfdb9e9aceb4955/src/tactic/alias.lean#L85

### CODE REVIEW

**Comment by digama0** [comment]
Looking at the mathlib3 version, it doesn't look like it ever copies the documentation on the original declaration, but rather it uses the docstring *on the alias command*, and if there is no doc on the alias command then it uses a description like ``**Alias** of `{n}`.``.

**Comment by dwrensha** [comment]
Ah, good catch. I'll update this...

**Comment by dwrensha** [comment]
Updated, and added the <-> case as well.

File: Mathlib/Tactic/Alias.lean [hunk 1]
```diff
@@ -73,6 +73,24 @@ def appendNamespace (ns : Name) : Name → Name
...
+def Target.toName : Target → Name
+| Target.plain n => n
+| Target.forward n => n
+| Target.backwards n => n
+
+/-- The docstring for an alias. -/
+def Target.toString : Target → String
+| Target.plain n => f!"**Alias** of `{n}`.".pretty
+| Target.forward n => f!"**Alias** of the forward direction of `{n}`.".pretty
+| Target.backwards n => f!"**Alias** of the reverse direction of `{n}`.".pretty
```
Comment (digama0) [inline Mathlib/Tactic/Alias.lean:92]:
```suggestion
| Target.plain n => s!"**Alias** of `{n}`."
| Target.forward n => s!"**Alias** of the forward direction of `{n}`."
| Target.backwards n => s!"**Alias** of the reverse direction of `{n}`."
```

Comment (dwrensha) [reply Mathlib/Tactic/Alias.lean:92]: Today I learned!

**Pull request closed without merging**